IBAnimatable/PresentationPresenterChacheKey.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  PresentationChacheKey.swift
+//  IBAnimatable
+//
+//  Created by Miro on 8/12/16.
+//  Copyright © 2016 IBAnimatable. All rights reserved.
+//
+
+import Foundation
+
+struct PresentationPresenterChacheKey: Hashable {
```

Comment:
I thought about that but could not find a proper way so far. I will have another look.